corehq/apps/users/models.py

```diff
@@ -1139,6 +1139,33 @@ def get(stale, raise_if_none):
         else:
             return None
 
+    @classmethod
+    @skippable_quickcache(['username'], skip_arg='strict')
+    def get_deleted_by_username(cls, username, strict=True):
+        def get(stale, raise_if_none):
+            result = cls.get_db().view('deleted_users/by_username',
```

Comment:
the convention for new code is to put anything that accesses a couch view in a function in a `dbaccessors` file. (sorry forgot to mention this earlier)